ci/tasks/run-int.sh

```diff
@@ -77,6 +70,15 @@ export ZONE=${google_zone}
 export GOPATH=${PWD}/bosh-cpi-src
 export PATH=${GOPATH}/bin:$PATH
 
+release_go_version="$(cat "$GOPATH/packages/golang/spec" | \
```

Comment:
awesome!!! 

now for nit: can you remove this duplication? either just run it as part of UTs or move it  to a function in `bosh-cpi-src/ci/tasks/utils.sh`